config/provider.go

```diff
@@ -294,6 +294,13 @@ func GetProvider() *ujconfig.Provider {
 			Type:          referenceType(pc, "project", "v1alpha1", "Project"),
 			TerraformName: "digitalocean_project",
 		}
+
+		r.References = map[string]ujconfig.Reference{
+			"forwarding_rule.certificate_name": {
+				TerraformName: "digitalocean_certificate",
+			},
+		}
```

Comment:
```suggestion
		r.References["forwarding_rule.certificate_name"] = map[string]ujconfig.Reference{
			: {
				TerraformName: "digitalocean_certificate",
			},
		}
```

This was the breaking change

Reply:
yeah i overseen the other reference configs for this resource